### DISCUSSION TITLE
backup from user account

### DISCUSSION THREAD

**Discussion by AxelJPG** [pocketbase/pocketbase#5872] [Q&A]
`It` is possible to create a backup with a user from the user collection

`
routerAdd('POST', '/test/:testId', (c) => {
  try {
    const name = 'test.zip'
    $app.createBackup(ctx, name)
  } catch (error) {
    c.json(400, { error })
  }
})
`

**Comment by ganigeorgiev** [comment]
Neither the title, nor the description and its code sample make sense. Please try to be more specific and elaborate more clearly what actually is the use case and what you are trying to do.

**Reply by AxelJPG** [reply]
Enable programmatic database backups for users with specific roles. Currently, this feature is restricted to the administrator. The question is whether there is an alternative way to achieve this, such as additional configurations or specific extensions, to delegate this ability without relying solely on the administrator.

**Reply by ganigeorgiev** [reply]
Again, it is still not clear what is the end goal here. The generated PocketBase backup is a full copy of your `pb_data`, including the admins accounts, SMTP password, OAuth2 credentials, etc. and allowing regular user to trigger and have access to them would be wrong.

If you want a regular user to generate and access a backup with only a subset of the available data, then this is not supported out of the box.
